bindings/pyroot/pythonizations/python/ROOT/_pythonization/_roofit/_rooworkspace.py

```diff
@@ -11,6 +11,29 @@
 
 from ._utils import _kwargs_to_roocmdargs, cpp_signature
 
+def make_json_for_variable(var_name, var_dict):
+    val, max_value, min_value, is_constant = [None, None, None, True]
+    if "value" in var_dict:
+        val = var_dict["value"]
+    if ("max" in var_dict) and ("min" in var_dict):
+        max_value, min_value, is_constant = var_dict["max"], var_dict["min"], False
+    if not (val is None) or not ((max_value is None) and (min_value is None)):
```

Comment:
Here, you can avoid one layer of nesting by instead of checking for the case where you have no `Error()`, you check for the case where you want to raise your error. Then, all the logic that follows here does not to be in any if statement, it will just run when the Error was not raised.